### PR TITLE
Added --version and --help

### PR DESCRIPTION
**Pull request by pzmudzinski**

### CODE REVIEW

File: .idea/vcs.xml [hunk 1]
```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (pavloo) [inline .idea/vcs.xml]: let's add `.idea` to `.gitignore`

File: version.py [hunk 2]
```diff
@@ -0,0 +1 @@
+__version__ = "0.0.1"
```
Comment (pavloo) [inline version.py]: let's aim `0.1.0` for the release

File: config_py/bin/config_py.py [hunk 3]
```diff
@@ -47,14 +52,23 @@ def generate_module_config(module):
...
+def print_version(ctx, param, value):
+    if not value or ctx.resilient_parsing:
+        return
+    click.echo(__version__)
+    ctx.exit()
+
+
+@click.command(context_settings=CONTEXT_SETTINGS)
 @click.option("-m", "--module", help="python module to create a config for", default=None)
+@click.option('-v', '--version', is_flag=True, callback=print_version,
```
Comment (pavloo) [inline config_py/bin/config_py.py]: let's add some tests for `-v` flag

Comment (pavloo) [reply config_py/bin/config_py.py]: where you read version from `fixtures/version.py`

**Review by pavloo — COMMENTED** [review]
cool 8)

File: config_py/bin/config_py.py [hunk 4]
```diff
@@ -16,6 +15,12 @@
     os.path.dirname(os.path.realpath(__file__)), '..', 'fixtures', 'module'
 )
 
+CONTEXT_SETTINGS = dict(help_option_names=['-h', '--help'])
+
+exec(open(os.path.join(
```
Comment (pavloo) [inline config_py/bin/config_py.py]: @pzmudzinski just asking, but is possible to just `import version` instead of doing this `exec`?

Comment (pzmudzinski) [reply config_py/bin/config_py.py]: This is for getting __version__ global variable, import wouldn't work here

Comment (pavloo) [reply config_py/bin/config_py.py]: do we really need it **global**?

Comment (pzmudzinski) [reply config_py/bin/config_py.py]: https://www.python.org/dev/peps/pep-0396/
"When a module (or package) includes a version number, the version SHOULD be available in the __ version __ attribute."

**Comment by pavloo** [comment]
addresses https://github.com/pavloo/config_py/issues/13 and https://github.com/pavloo/config_py/issues/10

**Pull request merged**